app/lib/dynamicRequire.js

```diff
@@ -117,4 +117,5 @@ module.exports = {
   'views/teachers/RestrictedToTeachersView': function () { return import(/* webpackChunkName: "RestrictedToTeachersView" */ 'views/teachers/RestrictedToTeachersView') },
   'views/courses/RestrictedToStudentsView': function () { return import(/* webpackChunkName: "RestrictedToStudentsView" */ 'views/courses/RestrictedToStudentsView') },
   'views/user/CertificatesView': function () { return import(/* webpackChunkName: "CertificatesView" */ 'views/user/CertificatesView') },
+  'views/LicensorView': function() { return import(/* webpackChunkName: "admin" */ 'views/LicensorView')}
```

Comment:
Should we be putting this in the admin chunk, rather than something like LicensorView chunk as line above?

Reply:
yes, we should create a separate chunk for this, was just testing with admin chunk, forgot to change that.